v3/fleet-local/control/chronos@.service

```diff
@@ -29,6 +29,7 @@ ExecStartPre=-/usr/bin/docker rm chronos
 ExecStart=/usr/bin/sh -c "docker run \
   --name chronos \
   --net=host \
+  --read-only \
```

Comment:
https://benchmarks.cisecurity.org/tools2/docker/CIS_Docker_1.11.0_Benchmark_v1.0.0.pdf search for ```5.12 Mount container's root filesystem as read only (Scored)```